samples/java/play-authenticate-usage/test/test/OAuth2Test.java

```diff
@@ -44,16 +42,11 @@ protected Application provideApplication() {
 
         amendConfiguration(additionalConfiguration);
 
```

Comment:
Removing this section is not related to the issue (it was not used obviously) - just found it while fixing `additionalConfiguration`.